install_files/ansible-base/roles/restrict_direct_access_app/templates/app_rules_v4

```diff
@@ -4,13 +4,8 @@
 :OUTPUT ACCEPT [0:0]
 :LOGNDROP - [0:0]
 
-# Drop and log all packets with invalid state.
--A OUTPUT -m state --state INVALID -j LOGNDROP
--A INPUT -m state --state INVALID -j LOGNDROP
-
 # Allow the tor instances used for ssh, source and document interface outbound access
 # Load before tor user drop rules
--A OUTPUT -p tcp -m owner --uid-owner messagebus -m state --state NEW,ESTABLISHED,RELATED -j ACCEPT -m comment --comment "tor instance that provides ssh access"
```

Comment:
This was left over from when the the source and document interface were in separate chroot jails. The user that the host system would see outside of the chroot jails was the messagebus user. This rule is not needed anymore with current non-chroot architecture